app/models/smart_proxy_feature.rb

```diff
@@ -0,0 +1,11 @@
+class SmartProxyFeature < ApplicationRecord
+  belongs_to :smart_proxy
+  belongs_to :feature
+  validates :feature, :uniqueness => {:scope => :smart_proxy_id}
+
+  store :settings, coder: JSON
+
+  def details
+    {settings: settings, capabilities: capabilities}
+  end
+end
```

Comment:
This model is rather empty, I wonder if we could move some of the code from smart-proxy here. Like *import* methods would be a good fit for starters.

Reply:
No insist, that's fine.